backend-project/small_eod/institutions/tests/test_validators.py

```diff
@@ -81,3 +88,24 @@ def test_regon_length(self):
 
         form = f(data=dict(regon=chars_14))
         self.assertTrue(form.is_valid())
+
+
+class InstitutionValidatorsTestCase(TestCase):
+    def test_level_3(self):
+        """
+        Administrative unit must be a level 3 unit.
+        """
+        category1 = CategoryFactory(level=1)
+        administrative_unit = JednostkaAdministracyjnaFactory(category=category1)
```

Comment:
```suggestion
        administrative_unit = JednostkaAdministracyjnaFactory(category__level=1)
```
wystarczy w taki sposób zapisać

Reply:
Zrobione. To samo w InstitutionSerializerTestCase.setUp